ui/panels-plugin/src/plugins/time-series-chart/index.ts

```diff
@@ -12,4 +12,10 @@
 // limitations under the License.
 
 export * from './TimeSeriesChart';
-export type { TimeSeriesChartDefinition } from './time-series-chart-model';
+export type {
+  TimeSeriesChartDefinition,
+  TimeSeriesChartOptions,
+  YAxisOptions,
+  PaletteOptions,
+  VisualOptions,
```

Comment:
Similar comment as above about giving these more verbose/prefixed names for clarity and to avoid conflicts with other panels.

Reply:
I imagine we will be able to share some of these with other panel types in the future, in particular `PaletteOptions` could be common with the future potential BarChart. Adding the prefixes though we can revisit, at that point we'd probably want to move them to @perses-dev/core anyways